src/Psl/Result/passthrough.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Result;
+
+/**
+ * This method can be used in combination with the `ResultInterface::proceed()` method
+ * inside the `$ifSucceeded` callback.
+ * It wil return the result value without any manipulation.
+ *
+ * @template T
+ * @psalm-return callable(T): T
+ * @psalm-pure
+ */
+function passthrough(): callable
+{
+    return static fn ($result) => $result;
+}
```

Comment:
hm, I think this better go inside the `Fun` namespace, since it doesn't have anything to do with `Result` actually. ( maybe `Fun\pass()` ? )